lib/numorstring/float32orstring.go

```diff
@@ -34,7 +33,20 @@ type Float32OrString struct {
 func (f *Float32OrString) UnmarshalJSON(b []byte) error {
```

Comment:
will update.  I'll remove the Float32OrString since we don't use it, and update Order to be a float32